### PR TITLE
feat: add logs to db replay in messenger

### PR DESCRIPTION
**Pull request by aeddi**
Signed-off-by: aeddi <antoine.e.b@gmail.com>

### CODE REVIEW

File: go/pkg/bertymessenger/util.go [hunk 1]
```diff
@@ -2,17 +2,23 @@ package bertymessenger
...
 	"bytes"
+	"context"
 	"encoding/base64"
 
+	"berty.tech/berty/v2/go/pkg/bertyprotocol"
 	"berty.tech/berty/v2/go/pkg/bertytypes"
 	"berty.tech/berty/v2/go/pkg/errcode"
 )
 
 func (svc *service) checkIsMe(gme *bertytypes.GroupMessageEvent) (bool, error) {
```
Comment (moul) [inline go/pkg/bertymessenger/util.go:13]: ambiguous, can you keep only one?

File: go/pkg/bertymessenger/util.go [hunk 2]
```diff
@@ -24,8 +30,12 @@ func (svc *service) checkIsMe(gme *bertytypes.GroupMessageEvent) (bool, error) {
 }
 
 func (svc *service) groupPKFromContactPK(contactPK []byte) ([]byte, error) {
```
Comment (moul) [inline go/pkg/bertymessenger/util.go:32]: same here, keep only one

**Review by moul — APPROVED** [review]
except for the two small details, I really like the way it's done :)

File: go/pkg/bertymessenger/util.go (hunk 1 above)
Comment (aeddi) [reply go/pkg/bertymessenger/util.go:13]: Yes I hesitated while doing it, I will change it ;)

File: go/pkg/bertymessenger/util.go (hunk 2 above)
Comment (aeddi) [reply go/pkg/bertymessenger/util.go:32]: 👍

**Pull request merged**